internal/build/docker.go

```diff
@@ -95,6 +99,42 @@ func (l *localDockerBuilder) buildBaseWithMounts(ctx context.Context, baseDocker
 	return getDigestFromOutput(output.String())
 }
 
+func (l *localDockerBuilder) buildImageWithSteps(ctx context.Context, baseDigest digest.Digest, tag string, steps []Cmd) (digest.Digest, error) {
+	imageWithSteps := baseDigest
+	for _, s := range steps {
+		resp, err := l.dcli.ContainerCreate(ctx, &container.Config{
+			Image: string(baseDigest),
```

Comment:
but also if you do it this way, this should be `imageWithSteps` lol

Reply:
yes good catch! 😅